City-of-Helsinki/drupal-hdbt
# UHF-5666 support icons in subtheme css

# Support icons in subtheme css (part of frontpage new component task) [UHF-5666](https://helsinkisolutionoffice.atlassian.net/browse/UHF-5666)


## What was done
* Changed relative path of hdbt icons that get referenced in subtheme to point to hdbt path instead of subtheme path

## How to install
* Make sure your instance is up and running on latest dev branch.
    * `git pull origin dev`
    * `make fresh`
* Update the HDBT theme
    * `composer require drupal/hdbt:dev-UHF-5666_support_icons_in_subtheme_css`
* Run `make drush-cr`

## How to test
* Without using this branch, try using a pseudo-icon in subtheme
* Get this branch
* Try using pseudo-icon in subtheme, it should now work
